### ISSUE TITLE
Collaborate with State of JS server

### ISSUE THREAD

**Issue by wesleytodd** [nodejs/next-10#241]
https://github.com/Devographics/surveys/issues/226

Thoughts on collaborating on this? cc @nodejs/next-10

**Comment by mhdawson** [comment]
I think it would be good to collaborate, sounds like back end was dropped from the existing survey which is too bad.

**Comment by wesleytodd** [comment]
Agreed, also too bad to see the folks piling on in the issue with non-additive comments. @SachaG if you want to have a more low key conversation maybe we can do it here? I think the original point of the issue would be a great conversation to have, and since this group has also put together a few surveys I think folks have good experience to share.

**Comment by SachaG** [comment]
Sounds good, we can get things started here.
